lib/pacemaker/pcmk_sched_actions.c

```diff
@@ -631,7 +631,7 @@ pcmk__update_action_for_orderings(pe_action_t *then, pe_working_set_t *data_set)
                          "Disabled ordering %s then %s in favor of %s then %s",
                          other->action->uuid, then->uuid, first->uuid,
                          then->uuid);
-            other->type = pcmk__ar_none;
+            other->type = (enum pe_ordering) pcmk__ar_none;
         }
```

Comment:
clang is also complaining about the use at lib/pacemaker/pcmk_sched_ordering.c:1115 for me.

Reply:
whoops, thanks